tests/unit/models/test_redditors.py

```diff
@@ -9,3 +11,29 @@ def test_search__params_not_modified(self):
         generator = self.reddit.redditors.search(None, params=params)
         assert generator.params["dummy"] == "value"
         assert params == {"dummy": "value"}
+
+    def test_user_data_by_account_ids(self):
+        with mock.patch.object(self.reddit, "request") as mock_method:
```

Comment:
It tests that the requests are well formed up to and over 100 IDs, since they need to be done in chunks of 100.

Failure of this test is meaningful because if you’re going to change the implementation to do something differently then you should have a good reason to make the change, since there is only one preferable and obviously correct implementation as far as I can tell.